dandischema/datacite.py

```diff
@@ -87,7 +90,13 @@ def to_datacite(
     attributes["descriptions"] = [
         {"description": meta.description, "descriptionType": "Abstract"}
     ]
-    attributes["publisher"] = "DANDI Archive"
+    attributes["publisher"] = {
+        "name": "DANDI Archive",
+        "schemeUri": "https://scicrunch.org/resolver/",
+        "publisherIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017571",
```

Comment:
is this really the identifier? or just `SCR_017571`

Reply:
in all of the examples https://datacite-metadata-schema.readthedocs.io/en/4.5/properties/publisher/#a-publisheridentifier they used a URL, so I just followed the trend... it is indeed quite an odd schema